res/res_pjsip_sdp_rtp.c

```diff
@@ -1956,6 +2018,38 @@ static int create_outgoing_sdp_stream(struct ast_sip_session *session, struct as
 		}
 	}
 
+	/* Copy any non asterisk formats to the rx side */
```

Comment:
I don't understand why this needs to occur, and also codecs should be treated as opaque

Reply:
I don't think we do, but I want to test some odd scenarios first.

Reply:
Is this still needed?